src/core/path.c

```diff
@@ -35,10 +35,7 @@ QuicPathInitialize(
     Path->EcnValidationState =
         Connection->Settings.EcnEnabled ? ECN_VALIDATION_TESTING : ECN_VALIDATION_FAILED;
 
-    if (MsQuicLib.ExecutionConfig &&
-        MsQuicLib.ExecutionConfig->Flags & QUIC_EXECUTION_CONFIG_FLAG_QTIP) {
-        CxPlatRandom(sizeof(Path->Route.TcpState.SequenceNumber), &Path->Route.TcpState.SequenceNumber);
-    }
+    CxPlatRandom(sizeof(Path->Route.TcpState.SequenceNumber), &Path->Route.TcpState.SequenceNumber);
```

Comment:
We shouldn't need to do this if we're not using QTIP. Please add back the `if` and fix it up to check if the setting was configured for QTIP.

Reply:
Fixed.